bHY6yh-d46940/cod-pol7d7_bHY6yh-d46940.xml

```diff
@@ -0,0 +1,250 @@
+            <author ref="#per-c7y65a">Geremia da Montagnone</author>
+            <respStmt>
+               <name ref="#jeffreycwitt">Jeffrey C. Witt</name>
+               <resp>Transcription Editor</resp>
+               <resp>TEI Encoder</resp>
+            </respStmt>
+            <respStmt>
+              <name>Katrina Van Der Ahe</name>
+              <resp>Transcription Editor</resp>
+              <resp>TEI Encoder</resp>
+            </respStmt>
+         </titleStmt>
+         <editionStmt>
+            <edition n="0.0.0-dev">
+               <title>Rubrica 6 [Krakow, lat 700 III Transcription]</title>
+               <date when="2023-06-26">June 26, 2023</date>
+            </edition>
+         </editionStmt>
+         <publicationStmt>
+            <authority>SCTA</authority>
+            <availability status="free">
+               <p>Published under a <ref target="https://creativecommons.org/licenses/by-nc-sa/4.0/">Attribution-NonCommercial-ShareAlike 4.0 International (CC BY-NC-SA 4.0)</ref>
+               </p>
+            </availability>
+         </publicationStmt>
+         <sourceDesc>
+            <listWit>
+               <witness xml:id="K" n="cod-pol7d7">Krakow Biblioteka Jagiellonska, MS lat. 700 III (CC.VIII.25)</witness>
+            </listWit>
+         </sourceDesc>
+      </fileDesc>
+      <encodingDesc>
+         <schemaRef n="lbp-diplomatic-1.0.0" url="https://raw.githubusercontent.com/lombardpress/lombardpress-schema/1.0.0/src/out/diplomatic.rng"/>
+         <editorialDecl>
+            <p>Encoding of this text has followed the recommendations of the LombardPress 1.0.0
+       guidelines for a diplomatic edition.</p>
+         </editorialDecl>
+      </encodingDesc>
+      <revisionDesc status="draft">
+         <listChange>
+            <change when="2021-05-18" status="draft" n="0.0.0">
+               <p>Created file for the first time.</p>
+            </change>
+         </listChange>
+      </revisionDesc>
+   </teiHeader>
+   <text xml:lang="la">
+      <front>
+         <div xml:id="starts-on">
+            <pb ed="#K" n="103-r"/>
+            <cb ed="#K" n="a"/>
+         </div>
+      </front>
+      <body>
+         <div xml:id="bHY6yh-d46940"><!-- p3l1-r6 -->
+            <head xml:id="bHY6yh-d46940-Hd1e108">Rubrica 6</head>
+           <head xml:id="bHY6yh-d46940-Hd1e111" type="question-title">De imaginatione: memoria: reminiscentia: et obliuione: consideratione morali</head>
+            <p xml:id="bHY6yh-d46940-d1e114">
+              <lb ed="#K"/>De imagina<lb ed="#K" break="no"/>tion 
+              memoria reminiscenti<lb ed="#K" break="no"/>a 
+              et obliuione morali considera<lb ed="#K" break="no"/>tione. 
+              Rubrica. 6
+            </p>
+            <p xml:id="bHY6yh-d46940-d1e123">
+              <lb ed="#K"/> <ref xml:id="bHY6yh-d46940-Rd1e119">PLato in timeo circa princi.</ref> <quote xml:id="bHY6yh-d46940-Qd1e129">Certum ilud 
```

Comment:
if "ilud" is really what's in the manuscript, this should be `<sic>ilud</sic>` since it is clearly incorrect. 

Could it actually be "illud"???


```suggestion
              <lb ed="#K"/> <ref xml:id="bHY6yh-d46940-Rd1e119">PLato in timeo circa princi.</ref> <quote xml:id="bHY6yh-d46940-Qd1e129">Certum ilud 
```

Reply:
I will check it when I go through and do my edits of this text

Reply:
I'm going to accept as is. Submit a new PR after you've gone back through :)